jsf/subsystem/src/main/java/org/jboss/as/jsf/deployment/JSFDependencyProcessor.java

```diff
@@ -121,7 +121,7 @@ private void addJSFAPI(String jsfVersion, ModuleSpecification moduleSpecificatio
         moduleSpecification.addSystemDependency(jsfAPI);
     }
 
-    // Is Jakarta Server Faces spec greater than 1.1?  If we add JSF 1.1 support we'll need this to keep from calling addJSFInjection()
+    // Is Jakarta Server Faces spec greater than 1.1?  If we add Faces 1.1 support we'll need this to keep from calling addJSFInjection()
```

Comment:
@fjuma Should these references to 1.1 be deleted or updated?

Reply:
I am doing this for https://github.com/jamezp/wildfly/tree/docs-test